app/models/search_vector.rb

```diff
@@ -49,6 +49,7 @@ def index!(discussion_ids)
     end
     handle_asynchronously :index!
   end
+  EventBus.listen('comment_create', 'comment_update') { |comment| index! comment.discussion_id }
```

Comment:
Updated the event bus to allow it to accept several event names at the same time.